src/Pages/HistoryPage/HistoryPage.jsx

```diff
@@ -0,0 +1,35 @@
+import { Loader, VideoCard } from "Components";
```

Comment:
Nothing in history, please check

Reply:
can you check again for me both preview link and the jsx page is working

Reply:
can you check again both live preview and jsx file is showing for me

Reply:
i checked, website is not working from my end